tools/corto/src/cortotool_build.c

```diff
@@ -293,7 +357,9 @@ corto_int16 cortotool_coverage(int argc, char *argv[]) {
 
 corto_int16 cortotool_rebuild(int argc, char *argv[]) {
 
-    cortotool_generateRakefile();
+    if (cortotool_clean(argc, argv)) {
+        goto error;
+    }
 
```

Comment:
This is a silent merge-conflict. Please remove the lines that explicitly do corto-clean.